source/llassetgen/source/FntWriter.cpp

```diff
@@ -0,0 +1,215 @@
+#include <fstream> 
+#include <float.h>
+#include <string>
+#include <vector>
+#include <cassert>
+#include <iostream>
+
+#include <llassetgen/FntWriter.h>
+#include <llassetgen/Image.h>
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+#include FT_OUTLINE_H
+
+namespace llassetgen {
+	FntWriter::FntWriter(FT_Face _face, std::string _face_name, int _font_size) {
+		face_name = _face_name;
+		face = _face;
+		font_info = Info();
+		font_common = Common();
+		char_infos = std::vector<CharInfo>();
+		kerning_infos = std::vector<KerningInfo>();
+		font_info.size = _font_size;
+		max_y_bearing = 0;
+	}
+
+	void FntWriter::setFontInfo() {
+		// collect font_info 
+		font_info.face = face->family_name + std::string(" ") + face->style_name;
+		font_info.is_bold == false;
+		font_info.is_italic == false;
+		if (face->style_flags == FT_STYLE_FLAG_ITALIC) {
+			font_info.is_italic == true;
+		}
+		if (face->style_flags == FT_STYLE_FLAG_BOLD) {
+			font_info.is_bold == true;
```

Comment:
Some of these `==` should be `=`

Reply:
Even the compiler printed some warnings about that :D